docs/standard/base-types/best-practices.md

```diff
@@ -204,73 +208,73 @@ The regular expression language in .NET includes the following language elements
 
 |Language element|Description|
 |----------------------|-----------------|
-|`(?=` `subexpression` `)`|Zero-width positive lookahead. Look ahead of the current position to determine whether `subexpression` matches the input string.|
-|`(?!` `subexpression` `)`|Zero-width negative lookahead. Look ahead of the current position to determine whether `subexpression` does not match the input string.|
-|`(?<=` `subexpression` `)`|Zero-width positive lookbehind. Look behind the current position to determine whether `subexpression` matches the input string.|
-|`(?<!` `subexpression` `)`|Zero-width negative lookbehind. Look behind the current position to determine whether `subexpression` does not match the input string.|
+|`(?=` `subexpression` `)`|Zero-width positive lookahead. Looks ahead of the current position to determine whether `subexpression` matches the input string.|
+|`(?!` `subexpression` `)`|Zero-width negative lookahead. Looks ahead of the current position to determine whether `subexpression` doesn't match the input string.|
+|`(?<=` `subexpression` `)`|Zero-width positive lookbehind. Looks behind the current position to determine whether `subexpression` matches the input string.|
+|`(?<!` `subexpression` `)`|Zero-width negative lookbehind. Looks behind the current position to determine whether `subexpression` doesn't match the input string.|
 
 ## Use time-out values
 
-If your regular expressions processes input that nearly matches the regular expression pattern, it can often rely on excessive backtracking, which impacts its performance significantly. In addition to carefully considering your use of backtracking and testing the regular expression against near-matching input, you should always set a time-out value to ensure that the impact of excessive backtracking, if it occurs, is minimized.
+If your regular expressions processes input that nearly matches the regular expression pattern, it can often rely on excessive backtracking, which impacts its performance significantly. In addition to carefully considering your use of backtracking and testing the regular expression against near-matching input, you must always set a time-out value to minimize the effect of excessive backtracking, if it occurs.
```

Comment:
```suggestion
If your regular expressions processes input that nearly matches the regular expression pattern, it can often rely on excessive backtracking, which impacts its performance significantly. In addition to carefully considering your use of backtracking and testing the regular expression against near-matching input, you should always set a time-out value to minimize the effect of excessive backtracking, if it occurs.
```